src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -220,11 +220,9 @@ public Task GetSubscriptionState()
             }
         }
 
-        [RavenAction("/databases/*/subscriptions/connection-details", "GET", AuthorizationStatus.ValidUser)]
+        [RavenAction("/databases/*/subscriptions/connection-details", "GET", AuthorizationStatus.ValidUser, CorsMode = CorsMode.Cluster)]
```

Comment:
When are we accessing this across the cluster?

Reply:
from ongoing tasks in studio - we always query responsible node directly.